geoips/yaml_configs/product_params/pmw_37/37pct.yaml

```diff
@@ -1,16 +1,16 @@
 37pct:
     product_type: interp_alg_cmap
-    alg_func: pmw_tb.pmw_37pct
+    alg_func: pmw_37pct
```

Comment:
@jsolbrig Also updated algorithms.  Note identical names for colormaps and algorithms.

Reply:
Great, looks good again!